test/tests/benchmark/ansible_control.py

```diff
@@ -0,0 +1,111 @@
+import os
+import time
+import re
+
+from modules.logger import Log
+
+from config.benchmark_config import *
+from ansible.playbook import PlayBook
+from ansible.inventory import Inventory
+from ansible import callbacks
+from ansible import utils
+from ansible import constants
+
+import jinja2
```

Comment:
Does the `jinja2` need a pre installation? If yes, you may add to test/requirements.txt